src/core/issues-notes.js

```diff
@@ -159,23 +136,58 @@ function handleIssues(ins, ghIssues, conf) {
         body = hyperHTML`${ghIssue.body_html}`;
       }
       div.append(body);
-      const level = parents(tit, "section").length + 2;
-      tit.setAttribute("aria-level", level);
+      const level = parents(titleParent, "section").length + 2;
+      titleParent.setAttribute("aria-level", level);
     }
     pub(report.type, report);
   });
   const issueSummaryElement = document.getElementById("issue-summary");
-  if (document.querySelectorAll(".issue").length) {
-    if (issueSummaryElement)
-      issueSummaryElement.innerHTML = issueSummary.innerHTML;
-  } else if (issueSummaryElement) {
-    pub("warn", "Using issue summary (#issue-summary) but no issues found.");
-    issueSummaryElement.remove();
+  if (issueSummaryElement) {
+    if (document.querySelectorAll(".issue").length) {
+      issueSummaryElement.append(...issueSummary.childNodes);
+    } else {
+      pub("warn", "Using issue summary (#issue-summary) but no issues found.");
+      issueSummaryElement.remove();
+    }
   }
 }
 
+/**
+ * @param {string} l10nIssue
+ * @param {Partial<Report>} report
+ */
+function createIssueSummaryEntry(l10nIssue, report, id) {
+  const issueNumberText = `${l10nIssue} ${report.number}`;
+  const title = report.title
+    ? hyperHTML`<span style="text-transform: none">: ${report.title}</span>`
+    : "";
+  return hyperHTML`
+    <li><a href="${"#" + id}">${issueNumberText}</a>${title}</li>
+  `;
+}
+
+/**
+ * @param {GitHubLabel[]} labels
+ * @param {string} title
+ * @param {string} repoURL
+ */
+function createLabelsGroup(labels, title, repoURL) {
+  const labelsGroup = Array.from(labels || [])
+    .map(label => {
```

Comment:
Move this into `createLabel` below?

Reply:
I think as we're just adding a `href: issuesURL.href` property here and using it just inside `createLabel`, it makes sense to move this into `createLabel` instead of using a `.map`. WDYT?

Reply:
Please open a new conversation rather than continuing outdated conversation.

Anyway, done! 😁